### DISCUSSION TITLE
Transfer Token to other account

### DISCUSSION THREAD

**Discussion by gogl92** [Laravel-Backpack/community-forum#394] [Q&A (Help)] [answered]
After finishing a project my client wants to have ownership of the purchased key, is there any way to transfer it to their account, I found that I can also change the email in my account as it's the only key I've purchased but I'd prefer an easier process.

**Comment by pxpm** [comment, marked as answer]
@gogl92 it should post no problem, send an email to `hello@backpackforlaravel.com` with the details and Cristian will help you.

Cheers
